
string[] cloud_names
sensor_msgs/PointCloud2[] clouds